src/styles.scss

```diff
@@ -0,0 +1,170 @@
+details[data-id="aital"] {
```

Comment:
Converted from `amITalkingALot.css`. Not sure how valuable the nesting is, but we want to mix in some variables.